src/Files.App/DataModels/NavigationControlItems/WslDistroItem.cs

```diff
@@ -5,30 +5,30 @@ namespace Files.App.DataModels.NavigationControlItems
 {
 	public class WslDistroItem : INavigationControlItem
 	{
-		public string Text { get; set; }
+		public string? Text { get; set; }
 
-		private string path;
+		private string? path;
 
-		public string Path
+		public string? Path
 		{
 			get => path;
 			set
 			{
 				path = value;
-				ToolTipText = Path.Contains('?', StringComparison.Ordinal) ? Text : Path;
+				ToolTipText = Path is not null ? (Path.Contains('?', StringComparison.Ordinal) ? Text : Path) : null;
 			}
 		}
 
-		public string ToolTipText { get; private set; }
+		public string? ToolTipText { get; private set; }
 
-		public NavigationControlItemType ItemType => NavigationControlItemType.LinuxDistro;
+		public NavigationControlItemType? ItemType => NavigationControlItemType.LinuxDistro;
 
-		public Uri Logo { get; set; }
+		public Uri? Logo { get; set; }
 
-		public SectionType Section { get; set; }
+		public SectionType? Section { get; set; }
 
-		public ContextMenuOptions MenuOptions { get; set; }
+		public ContextMenuOptions? MenuOptions { get; set; }
 
-		public int CompareTo(INavigationControlItem other) => Text.CompareTo(other.Text);
-	}
+        public int CompareTo(INavigationControlItem? other) => Text is not null ? Text.CompareTo(other?.Text) : 0;
+    }
```

Comment:
Formatting + double ternary.

Reply:
Is it a double ternary? (i may just be stupid)